src/core/aws-config.adb

```diff
@@ -583,16 +582,17 @@ package body AWS.Config is
       return To_String (O.P (Protocol_Family).Str_Value);
    end Protocol_Family;
 
-   ---------------------
-   -- Read_If_Present --
-   ---------------------
+   --------------------
+   -- Read_Or_Ignore --
+   --------------------
 
-   procedure Read_If_Present (Filename : String) is
+   procedure Read_Or_Ignore (Filename : String) is
    begin
-      if Directories.Exists (Filename) then
```

Comment:
I don't agree with this change as it uses exception as control flow.

Reply:
We still need to check for Directory.Exists() and catch all other exceptions if needed.

Reply:
Apart from that I'm ok with this change. We hide internal exception when reading ini file but still raise exception for explicit call to Read() from user code. Seem a good compromise.

Reply:
Ok. I'll restore check file existence inside of Read_Or_Ignore.